server/index.js

```diff
@@ -0,0 +1,24 @@
+const express = require('express');
+const db = require('../db/index.js');
+const Bundles = require('../db/models/bundles.model.js');
+
+const app = express();
+const port = 3663;
+
+db.sync({ force: false })
+  .then(() => {
+    console.log('db synced: ', db);
+    app.listen(port, () => {
+      console.log(`dockside at port ${port}`);
+    })
+  })
+  .then(() => {
+    Bundles.findOne({
+      where: {
+        id: 100,
+      },
+    })
+    .then((results) => {
+      console.log('results: ', results.dataValues);
+    });
+  });
```

Comment:
So when we run the server file it's not recreating the database but connecting then finding the last row and then logging it to the console. Makes sense to check that there is a row item of 'id' 100. 

My only suggestion would be to include a catch block for any errors. Possibly if findOne can't find 'id' 100.

Reply:
Yep, good point, will do that.